src/components/transport_manager/test/transport_manager_impl_test.cc

```diff
@@ -417,14 +414,15 @@ TEST_F(TransportManagerImplTest, Disconnect_DisconnectionFailed) {
   EXPECT_CALL(*mock_adapter_, Disconnect(mac_address_, application_id_))
       .WillOnce(Return(TransportAdapter::FAIL));
   // Assert
-  EXPECT_EQ(E_INTERNAL_ERROR, tm_.Disconnect(connection_key_));
+  // Even with fail, we get Success
```

Comment:
@AByzhynar This comment explains why with TransportAdapter::FAIL we expect Success - that is the current logic in GENIVI